easybuild/easyconfigs/i/IPython/IPython-5.8.0-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,244 @@
+    }),
+    ('futures', '3.2.0', {
+        'modulename': 'concurrent.futures',
+        'source_urls': ['https://pypi.python.org/packages/source/f/futures/'],
+        'checksums': ['9ec02aa7d674acb8618afb127e27fde7fc68994c0437ad759fa094a574adb265'],
+    }),
+    ('functools32', '3.2.3-2', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/functools32/'],
+        'checksums': ['f6253dfbe0538ad2e387bd8fdfd9293c925d63553f5813c4e587745416501e6d'],
+    }),
+    ('ptyprocess', '0.6.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/ptyprocess/'],
+        'checksums': ['923f299cc5ad920c68f2bc0bc98b75b9f838b93b599941a6b63ddbc2476394c0'],
+    }),
+    ('simplegeneric', '0.8.1', {
+        'source_tmpl': 'simplegeneric-%(version)s.zip',
+        'source_urls': ['https://pypi.python.org/packages/source/s/simplegeneric/'],
+        'checksums': ['dc972e06094b9af5b855b3df4a646395e43d1c9d0d39ed345b7393560d0b9173'],
+    }),
+    ('scandir', '1.9.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/scandir/'],
+        'checksums': ['44975e209c4827fc18a3486f257154d34ec6eaec0f90fef0cca1caa482db7064'],
+    }),
+    ('pathlib2', '2.3.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pathlib2/'],
+        'checksums': ['25199318e8cc3c25dcb45cbe084cc061051336d5a9ea2a12448d3d8cb748f742'],
+    }),
+    ('pickleshare', '0.7.5', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pickleshare/'],
+        'checksums': ['87683d47965c1da65cdacaf31c8441d12b8044cdec9aca500cd78fc2c683afca'],
+    }),
+    ('traitlets', '4.3.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/traitlets/'],
+        'checksums': ['9c4bd2d267b7153df9152698efb1050a5d84982d3384a37b2c1f7723ba3e7835'],
+    }),
+    ('parso', '0.3.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/parso/'],
+        'checksums': ['68406ebd7eafe17f8e40e15a84b56848eccbf27d7c1feb89e93d8fca395706db'],
+    }),
+    ('jedi', '0.13.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/j/jedi/'],
+        'checksums': ['b7493f73a2febe0dc33d51c99b474547f7f6c0b2c8fb2b21f453eef204c12148'],
+    }),
+    ('configparser', '3.5.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/c/configparser/'],
+        'checksums': ['5308b47021bc2340965c371f0f058cc6971a04502638d4244225c49d80db273a'],
+    }),
+    ('backports.shutil_get_terminal_size', '1.0.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/backports.shutil_get_terminal_size/'],
+        'checksums': ['713e7a8228ae80341c70586d1cc0a8caa5207346927e23d09dcbcaf18eadec80'],
+    }),
+    ('testpath', '0.4.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/testpath/'],
+        'checksums': ['b694b3d9288dbd81685c5d2e7140b81365d46c29f5db4bc659de5aa6b98780f8'],
+    }),
+    ('nose', '1.3.7', {
```

Comment:
@jose-d These easyconfigs include a lot of additional extensions compared to #7647, even things already included with `Python` itself, can you clarify why that is?

Reply:
it's long time ago already..., I'll check it.

Reply:
synced from 7647.
> These easyconfigs include a lot of additional extensions
there was just different order as both PRs were originally independent.
> even things
mhmm, I found just `certifi` and removed it from this PR.